app/models/search/work_search_form.rb

```diff
@@ -176,16 +176,16 @@ def search_results
   ###############
 
   SORT_OPTIONS = [
-    ['Best Match', '_score'],
-    ['Author', 'authors_to_sort_on'],
-    ['Title', 'title_to_sort_on'],
-    ['Date Posted', 'created_at'],
-    ['Date Updated', 'revised_at'],
-    ['Word Count', 'word_count'],
-    ['Hits', 'hits'],
-    ['Kudos', 'kudos_count'],
-    ['Comments', 'comments_count'],
-    ['Bookmarks', 'bookmarks_count']
+    ["Best Match", "_score"],
+    ["Creator", "authors_to_sort_on"],
+    ["Title", "title_to_sort_on"],
+    ["Date Posted", "created_at"],
+    ["Date Updated", "revised_at"],
+    ["Word Count", "word_count"],
+    ["Hits", "hits"],
+    ["Kudos", "kudos_count"],
+    ["Comments", "comments_count"],
```

Comment:
Wondering if I should actually do this, since we have instances of whitespace in "Date Posted", "Date Updated", etc. so it wouldn't be consistent on each? I'm a Ruby noob, welcome to other thoughts here.

Reply:
I personally don't love the mix of array styles, but we do it in the tag_search_form, so I guess it makes sense to do it here as well.